trax/tf_numpy/extensions/extensions_test.py

```diff
@@ -368,6 +370,33 @@ def f(a):
     check_trace_only_once(
         tf.convert_to_tensor(value=1), tf.convert_to_tensor(value=2))
 
+
```

Comment:
It's still good to test the internal function `_compose_output_rep`. Accessing internal symbols in an accompanying test file is acceptable.